R/PipeOpTuneThreshold.R

```diff
@@ -143,7 +143,12 @@ PipeOpTuneThreshold = R6Class("PipeOpTuneThreshold",
     },
     .task_to_prediction = function(input) {
       prob = as.matrix(input$data(cols = input$feature_names))
-      colnames(prob) = unlist(input$levels())
```

Comment:
this worked before because there was only one col in the task with factors for classification, but it is very brittle and should be avoided.